app/lib/features/news/pages/news_page.dart

```diff
@@ -26,14 +24,8 @@ class NewsPage extends ConsumerWidget {
         actions: <Widget>[
           Visibility(
             visible: canPostNews,
-            child: IconButton(
-              key: NewsUpdateKeys.addNewsUpdate,
+            child: PlusIconWidget(
```

Comment:
just a 3/10 feedback: looking at the video this being blue felt a bit much ... it isn't used that often and drawing that much attention on it felt a bit over the top...

I wonder if we might want a more subtle version of that PlusIconWidget that is less pungent.

Reply:
I was also thinking about it while developer.

I have change it with alpha value and this is how it looks now.

<img width="386" alt="Screenshot 2024-06-26 at 8 32 30 AM" src="https://github.com/acterglobal/a3/assets/51526480/c7ad75d1-edc5-41c5-8e9c-cbb9c25c9e3f">

<img width="379" alt="Screenshot 2024-06-26 at 8 32 41 AM" src="https://github.com/acterglobal/a3/assets/51526480/4b301d2e-5c08-4c81-a446-6b26c3c1b6ac">